composer.json

```diff
@@ -23,13 +23,13 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.1",
+        "php": "^7.1 || ^8.0",
         "prooph/common" : "^4.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0",
-        "phpspec/prophecy": "^1.7",
-        "prooph/php-cs-fixer-config": "^0.3",
+        "phpunit/phpunit": "^6.0 || ^9.3",
+        "phpspec/prophecy": "^1.9",
```

Comment:
Update Prophecy was necessary to avoid `Function ReflectionType::__toString() is deprecated` error during integration testing on PHP 7.4.

Reply:
:+1: